homeassistant/components/light/tellstick.py

```diff
@@ -5,126 +5,84 @@
 https://home-assistant.io/components/light.tellstick/
 """
 from homeassistant.components.light import ATTR_BRIGHTNESS, Light
-from homeassistant.const import EVENT_HOMEASSISTANT_STOP
-
+from homeassistant.components.tellstick import (TellstickDevice,
+                                                TELLCORE_REGISTRY,
```

Comment:
These variables are overwritten when `tellstick` is initialized. To prevent the tellstick light component from pointing at the initial value (`None`) you want to import the component and refer to the variables based on that.

```python
from homeassistant.components import tellstick

tellstick.TELLCORE_REGISTRY
```